File: backend/src/main/java/com/apocalypse/thefall/dto/TileDto.java
package com.apocalypse.thefall.dto;

import java.util.Map;

public record TileDto(
        PositionDto position,
        String type,
        boolean walkable,
        int movementCost,
        boolean revealed,
        boolean visible,
        Map<String, String> descriptions
) {
}

File: backend/src/main/java/com/apocalypse/thefall/dto/TileDto.kt
package com.apocalypse.thefall.dto;

import java.util.Map;

public record TileDto(
        Long id,
        PositionDto position,
        String type,
        boolean walkable,
        int movementCost,
        boolean revealed,
        boolean visible,
        Map<String, String> descriptions
) {
}
